transformers/src/unit-test/java/uk/gov/moj/sdt/transformers/BulkRequestTransformerTest.java

```diff
@@ -212,15 +211,15 @@ public void testTransformDomainToJaxb() {
 
         final BulkResponseType jaxb = transformer.transformDomainToJaxb(domain);
 
-        Assert.assertEquals("The Sdt Bulk Reference is as expected", domain.getSdtBulkReference(),
-                jaxb.getSdtBulkReference());
-        Assert.assertEquals("The customer reference is as expected", domain.getCustomerReference(),
-                jaxb.getCustomerReference());
-        Assert.assertEquals("The Sdt Service is as expected", AbstractTransformer.SDT_SERVICE, jaxb.getSdtService());
-        Assert.assertEquals("The number of requests are as expected", jaxb.getRequestCount(),
-                domain.getNumberOfRequest());
-        Assert.assertNotNull("The submitted date is found", jaxb.getSubmittedDate());
+        assertEquals(domain.getSdtBulkReference(),
+                jaxb.getSdtBulkReference(), "The Sdt Bulk Reference is as expected");
+        assertEquals(domain.getCustomerReference(), jaxb.getCustomerReference(),
+                "The customer reference is as expected");
+        assertEquals(AbstractTransformer.SDT_SERVICE, jaxb.getSdtService(), "The Sdt Service is as expected");
+        assertEquals(jaxb.getRequestCount(), domain.getNumberOfRequest(),
+                "The number of requests are as expected");
+        assertNotNull(jaxb.getSubmittedDate(), "The submitted date is found");
 
-        Assert.assertEquals("The status code is as expected", jaxb.getStatus().getCode().value(), "Ok");
+        assertEquals(jaxb.getStatus().getCode().value(), "Ok", "The status code is as expected");
```

Comment:
The text used for the messages in the assert statements seems to be the opposite to what it should be!  For example, I'd expect the text for the message displayed if the assertNotNull() fails to be "The submitted date is not found".

Reply:
assertEquals parameter order are now diff. The message is now the 3rd param, whereas before it was the 1st param. Painful transition to implement!